libs/ui-toolkit/src/components/background-video/background-video.tsx

```diff
@@ -1,11 +1,19 @@
-export const BackgroundVideo = () => {
+import classNames from 'classnames';
+
+type BackgroundVideoProps = {
+  className?: string;
+};
+export const BackgroundVideo = ({ className }: BackgroundVideoProps) => {
   return (
     <video
       autoPlay
       muted
       loop
       playsInline
-      className="absolute left-0 top-0 w-full h-full object-cover"
+      className={classNames(
+        'absolute left-0 top-0 w-full h-full object-cover',
+        className
+      )}
       poster="https://vega.xyz/poster-image.jpg"
```

Comment:
**nitpick (non-blocking)**: This should be re-hosted, so that a future site update doesn't accidentally break the layout/display of things on the console/explorer. Non-blocking as it wasn't added in this commit and should be a separate ticket